src/PaymentSlip.php

```diff
@@ -1453,10 +1453,27 @@ public function getAllElements()
                 $paymentSlipData->getPayerLine3(),
                 $paymentSlipData->getPayerLine4()
             ];
-            $elements['payerLeft'] = [
-                'lines' => $lines,
+            // need to separate, as left payer lines are much shorter
```

Comment:
Please extract that left and right wrapping related code into a protected method each. Its doc block should explain the what and why (in short). The code looks very similar, by the way, probably we need only one method, which can take the line length and cut parameters.